pkg/util/common.go

```diff
@@ -199,3 +199,92 @@ func TagStringToSemver(tag string) (semver.Version, error) {
 func SemverToTagString(tag semver.Version) string {
 	return AddTagPrefix(tag.String())
 }
+
+// CopyFile copies a file from one location to another.
+func CopyFile(src, dst string, ignoreNotExist bool) error {
+	srcStat, err := os.Stat(src)
+	if err != nil && !ignoreNotExist {
+		return err
+	}
+	if os.IsNotExist(err) && ignoreNotExist {
+		return nil
+	}
+
+	if !srcStat.Mode().IsRegular() {
+		return errors.New("file mode must be regular to be copied")
```

Comment:
Can we be really explicit about what "regular" means?

i.e., https://golang.org/pkg/os/#FileMode.IsRegular:
> IsRegular reports whether m describes a regular file. That is, it tests that no mode type bits are set.